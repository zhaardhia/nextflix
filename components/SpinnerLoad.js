import React from 'react'
import styles from '../styles/SpinnerLoad.module.css'

const SpinnerLoad = () => {
  return (
    <div className={[styles["grow-pulse"], styles.spinner].join(" ")}></div>
  )
}

export default SpinnerLoad