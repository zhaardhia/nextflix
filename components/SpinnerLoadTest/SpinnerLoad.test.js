import { render, screen } from "@testing-library/react";
import SpinnerLoad from "../SpinnerLoad";
import '@testing-library/jest-dom'


describe("Loader animation component", () => {
  it("renders successfully", () => {
    const { container } = render(<SpinnerLoad />);
    expect(container).toMatchSnapshot();
  });
});