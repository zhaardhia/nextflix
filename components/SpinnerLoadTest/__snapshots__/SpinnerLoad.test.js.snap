// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader animation component renders successfully 1`] = `
<div>
  <div
    class="grow-pulse spinner"
  />
</div>
`;
